OpenCL_Ext.txt

```diff
@@ -8,7 +8,7 @@ Khronos{R} OpenCL Working Group
 :data-uri:
 :icons: font
 :toc2:
-:toclevels: 1
+:toclevels: 2
```

Comment:
If we decide to keep one TOC level for the HTML output but use two TOC levels for the PDF output I think this will do it.

```suggestion
// Only one TOC level for the HTML output, two TOC levels for everything else.
ifdef::backend-html5[:toclevels: 1]
ifndef::backend-html5[:toclevels: 2]
```

Reply:
Thanks, I'll commit this.